### PR TITLE
ERC1155Timelock

### PR DESCRIPTION
**Pull request by anhnq82**

### CODE REVIEW

**Review by Zergity — CHANGES_REQUESTED** [review]
please remove unused packages in `packages.json`

and add our timelock tests from the -core repo

File: contracts/token/ERC1155/ERC1155.sol [hunk 1]
```diff
@@ -19,6 +22,7 @@ import "../../utils/introspection/ERC165.sol";
  */
 contract ERC1155 is Context, ERC165, IERC1155, IERC1155MetadataURI {
```
Comment (BerlinP) [inline contracts/token/ERC1155/ERC1155.sol]: I think we should use ERC1155Timelock instead of ERC1155 to avoid unnecessary confusion

**Review by BerlinP — CHANGES_REQUESTED** [review]
I think we should use ERC1155Timelock instead of ERC1155 to avoid unnecessary confusion

File: contracts/token/ERC1155/ERC1155.sol (hunk 1 above)
Comment (Zergity) [reply contracts/token/ERC1155/ERC1155.sol]: yes, we just have to make a separated commit for it

**Pull request closed without merging**